service/arm_templates/private_load_balancer_setup.json

```diff
@@ -105,6 +105,10 @@
         "id": "[variables('loadBalancerBackendPoolID')]",
         "name": "[variables('loadBalancerBackendPoolName')]"
       }
+    },
+    "loadBalancerIPAddress": {
```

Comment:
I'd like to unify those outputs and make it just `ipAddess`. It's quite obvious for private LB it'll be private IP and public IP for public LB. WDYT?